### PR TITLE
Run interval from config

### PR DESCRIPTION
**Pull request by myakove**

### CODE REVIEW

**Comment by myakove** [comment]
`tox` fixed in https://github.com/RedHatQE/ci-jobs-trigger/pull/98

File: config-examples/zstream-trigger-config.example.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ processed_versions_file_path: <path to processed versions file>
 # Optional
 slack_webhook_url: <slack webhook url to post job status>
 slack_errors_webhook_url: <slack webhook url to post code errors>
+run_interval: 24h # can be s/m/h
```
Comment (rnetser) [inline config-examples/zstream-trigger-config.example.yaml:8]: readme please

**Review by rnetser — CHANGES_REQUESTED** [review]

**Review by rnetser — APPROVED** [review]

**Comment by redhat-qe-bot1** [comment]
New container for quay.io/redhat_msi/ci-jobs-trigger:latest published

**Pull request merged**